pkg/usagestats/reporter.go

```diff
@@ -33,13 +32,31 @@ const (
 	defaultStatsServerURL      = "https://stats.grafana.org/mimir-usage-report"
 )
 
+const (
+	CustomInstallationMode  = "custom"
+	HelmInstallationMode    = "helm"
+	JsonnetInstallationMode = "jsonnet"
+)
```

Comment:
A couple of nits:
- We're used to use the same common prefix for constants. Also I don't think we need to expose them.
- You can define all supported values, so that in the CLI flag description you can do `fmt.Sprintf("Supported values: %s.", strings.join(supportedInstallationModes, ", "))`

```suggestion
const (
	installationModeCustom  = "custom"
	installationModeHelm    = "helm"
	installationModeJsonnet = "jsonnet"
)

var (
    supportedInstallationModes = []string{installationModeCustom, installationModeHelm, installationModeJsonnet}
)
```